charts/datadog/templates/_components-common-env.yaml

```
@@ -60,4 +60,8 @@
 - name: DD_EXCLUDE_PAUSE_CONTAINER
   value: "false"
 {{- end }}
+{{- if $.Values.datadog.securityAgent.runtime.remoteConfiguration.enabled }}
+- name: DD_REMOTE_CONFIGURATION_ENABLED
+  value: "true"
+{{- end }}
```

is it needed in every container?

---

It is required by the core agent container in order to turn on remote configuration. For other containers it provides an easy way to test whether remote config is enabled at the core-agent level.

---

since remove config will be used by different products. It can be good to create a function in `_helper.tpl` that will be use to know if the envvar needs to be added, something like 

```suggestion
{{- if eq (include "remote-configuation-enabled" .) "true" }}
- name: DD_REMOTE_CONFIGURATION_ENABLED
  value: "true"
{{- end }}
```

---

Thanks, I added the `should-enable-remote-configuration` function in `_helper.tpl`. Other products that want to enable remote configuration will just have to add an OR condition to this function.